swagger.yaml

```diff
@@ -1049,13 +1049,18 @@ components:
         roomNumber:
           type: string
           description: Appointment room number
+        alwaysCheckIn:
+          type: boolean
+          description: true if it is mandatory to sign up for the appointment at the desk
+
 
     AppointmentStatus:
       type: string
       enum:
         - AVAILABLE
         - LOCKED
         - RESERVED
+        - CHECKED_IN
```

Comment:
Remove, use the "Visited" field from DH appointment and add it to our appointment object